crates/tauri-bundler/src/bundle/windows/util.rs

```diff
@@ -70,8 +70,20 @@ pub fn download_webview2_offline_installer(base_path: &Path, arch: &str) -> crat
 pub fn download(url: &str) -> crate::Result<Vec<u8>> {
   log::info!(action = "Downloading"; "{}", url);
 
-  let agent = ureq::AgentBuilder::new().try_proxy_from_env(true).build();
-  let response = agent.get(url).call().map_err(Box::new)?;
+  let (agent, url) = match std::env::var("GITHUB_CDN") {
+    Ok(cdn) => {
+      let mut parsed_cdn = url::Url::parse(&cdn).expect("Invalid cdn");
```

Comment:
We should propagate the error using `?` instead of panicing with `.expect` or better use it with `.context()` from `anyhow` crate.

```suggestion
      let mut parsed_cdn = url::Url::parse(&cdn).context("invalid tools download host URL")?;
```

You will need to import `anyhow::Context` trait